ClosedXML_Tests/Excel/CalcEngine/FunctionsTests.cs

```diff
@@ -693,5 +693,16 @@ public void MultipleUnaryOperators(string formula, double expectedResult)
 
             Assert.AreEqual(expectedResult, res, XLHelper.Epsilon);
         }
+
+        [TestCase("RIGHT(\"2020\", 2) + 1", 21)]
```

Comment:
I would prefer not to explicitly coerce the evaluation results to `double`. I think rather leave the expected value as objects and let the assertions therefore also test the object type. i.e. something like this:

```c#
        [TestCase("RIGHT(\"2020\", 2) + 1", 21)]
        [TestCase("LEFT(\"20.2020\", 6) + 1", 21.202)]
        [TestCase("2 + (\"3\" & \"4\")", "36")]
        [TestCase("2 + \"3\" & \"4\"", "54")]
        [TestCase("\"3\" & \"4\"", 34)]
        public void TestStringSubExpression(string formula, object expectedResult)
        {
            var actual = XLWorkbook.EvaluateExpr(formula);

            Assert.AreEqual(expectedResult, actual);
        }
```
(one more test also included)

Reply:
The only problem is that without casting to double one cannot specify a tolerance that may cause troubles due to inexact comparison.

Reply:
But that doesn't seem to be a problem with these test cases.

Reply:
OK then

Reply:
@igitur I have made the requested changes to the test.

Reply:
@mathlang I don't see the changes.

Reply:
@igitur My bad, not sure what happened. They should be there now.